aidial_adapter_openai/constant.py

```diff
@@ -1,4 +1,9 @@
-from openai import Timeout
+import httpx
 
 # connect timeout and total timeout
-DEFAULT_TIMEOUT = Timeout(600, connect=10)
+DEFAULT_TIMEOUT = httpx.Timeout(600, connect=10)
+
+# Borrowed from openai._constants.DEFAULT_CONNECTION_LIMITS
+DEFAULT_CONNECTION_LIMITS = httpx.Limits(
+    max_connections=1000, max_keepalive_connections=100
```

Comment:
Minor:

Theoretically we could calculate `max_keepalive_connections` during startup of application, since http polling works that way:

```
When you make several requests to the same host, the Client will reuse the underlying TCP connection, instead of recreating one for every single request.
```

Theoretically, we know exact number of hosts, that we we will use from deployments

Reply:
That's a great idea. Let's leave it for another MR.